diff -ruN libusb-vanilla/libusb/os/linux_netlink.c libusb-modified/libusb/os/linux_netlink.c
--- libusb-vanilla/libusb/os/linux_netlink.c	2021-10-31 11:33:12.000000000 +0300
+++ libusb-modified/libusb/os/linux_netlink.c	2023-10-23 20:09:23.309671739 +0300
@@ -288,7 +288,6 @@
 	int detached, r;
 	ssize_t len;
 	struct cmsghdr *cmsg;
-	struct ucred *cred;
 	struct sockaddr_nl sa_nl;
 	struct iovec iov = { .iov_base = msg_buffer, .iov_len = sizeof(msg_buffer) };
 	struct msghdr msg = {
@@ -322,12 +321,6 @@
 		return -1;
 	}

-	cred = (struct ucred *)CMSG_DATA(cmsg);
-	if (cred->uid != 0) {
-		usbi_dbg(NULL, "ignoring netlink message with non-zero sender UID %u", (unsigned int)cred->uid);
-		return -1;
-	}
-
 	r = linux_netlink_parse(msg_buffer, (size_t)len, &detached, &sys_name, &busnum, &devaddr);
 	if (r)
 		return r;
