From 678cab799449a35e55f5a4d63c8f2450f905888d Mon Sep 17 00:00:00 2001
From: "alexey.lysiuk" <alexey.lysiuk@gmail.com>
Date: Tue, 26 Sep 2023 09:48:11 +0300
Subject: [PATCH] - fixed `DumpCPUInfo()` definition for non-x64 platforms

fix #2188
---
 src/common/utility/x86.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/utility/x86.cpp b/src/common/utility/x86.cpp
index 78d74fff9..5707c6572 100644
--- a/src/common/utility/x86.cpp
+++ b/src/common/utility/x86.cpp
@@ -45,7 +45,7 @@ void CheckCPUID(CPUInfo *cpu)
 	cpu->DataL1LineSize = 32;	// Assume a 32-byte cache line
 }
 
-FString DumpCPUInfo(const CPUInfo *cpu)
+FString DumpCPUInfo(const CPUInfo *cpu, bool brief)
 {
 	return FString();
 }
-- 
2.41.0

