diff --git a/wadsrc/static/zscript/actors/attacks.zs b/wadsrc/static/zscript/actors/attacks.zs
index 188a9c064..db9da6f35 100644
--- a/wadsrc/static/zscript/actors/attacks.zs
+++ b/wadsrc/static/zscript/actors/attacks.zs
@@ -626,7 +626,7 @@ extend class Actor
 	}
 
 	deprecated("2.3", "For Dehacked use only")
-	void A_RadiusDamage(int dam, double dist)
+	void A_RadiusDamage(int dam, int dist)
 	{
 		A_Explode(dam, dist);
 	}
