--- libappindicator-12.10.1+18.04.20180322.1/src/app-indicator.c	2018-03-22 01:15:43.000000000 +0000
+++ ../libappindicator/libappindicator-12.10.1+18.04.20180322.1/src/app-indicator.c	2020-03-21 23:06:34.879451825 +0000
@@ -1616,7 +1616,7 @@
 		gint n_elements, i;
 		gboolean found=FALSE;
 		gtk_icon_theme_get_search_path(icon_theme, &path, &n_elements);
-		for (i=0; i< n_elements || path[i] == NULL; i++) {
+		for (i=0; i< n_elements; i++) {
 			if(g_strcmp0(path[i], theme_path) == 0) {
 				found=TRUE;
 				break;
